node/node.go

```diff
@@ -1276,7 +1312,10 @@ func (app *App) startServices(ctx context.Context) error {
 		if app.Config.SMESHING.CoinbaseAccount == "" {
 			return fmt.Errorf("smeshing enabled but no coinbase account provided")
 		}
-		if err := app.postSupervisor.Start(app.Config.SMESHING.Opts); err != nil {
+		if len(app.signers) > 1 {
+			return fmt.Errorf("supervised smeshing cannot be started in a multi-smeshing setup")
+		}
```

Comment:
Primarily because it would introduce a lot of confusion and special handling of the supervised identity compared to the "remote" identities. See also my other comment. Supervised PoST exists primarily to ensure existing setups don't break - I think we should consider phasing it out eventually in some future update.